kgforge/specializations/stores/bluebrain_nexus.py

```diff
@@ -522,7 +543,7 @@ def _update_one(self, resource: Resource, schema_id: str) -> None:
             metadata_context=None,
             context_resolver=self.service.resolve_context
         )
-        url, params = self.service._prepare_uri(resource, schema_id)
+        url, params = self.service._prepare_uri(resource, schema_id, keep_unconstrained=True)
```

Comment:
is `keep_unconstrained` really necessary? also, where can the user modify it?

Reply:
_prepare_uri is used by most methods (register, tag, update...)
The issue is that in the case of update, and update schema, you'd like the unconstrained schema to be provided in the uri (to edit the schema for it to be unconstrained, or to keep it as is), but in the case of register, tag... you'd want an _, if ever the schema that is provided is unconstrained.json

Reply:
I had one last though, maybe naming it `use_unconstrained_id`, to be more specific that you want to use the id?

Reply:
Just changed it